third_party/amd/lib/TritonAMDGPUToLLVM/BuiltinFuncToLLVM.cpp

```diff
@@ -195,6 +196,16 @@ class CallOpConversion : public mlir::RewritePattern {
       LLVM::FastmathFlagsAttr defaultFlags{};
       replacementOp = rewriter.create<LLVM::FMulOp>(
           loc, returnType, operands[0], rcpOp->getResult(0), defaultFlags);
+    } else if (calleeName == "__triton_hip_fast_expf") {
+      assert(operands.size() == 1);
```

Comment:
Can you add some lit tests for this new pattern?

Reply:
Definitely. Where should I put these tests? I didn't find similar testcases in the codebase.

Reply:
Yup. we should add more lit tests in general. You can put a new file under `test/Conversion/amd/builtin_func_to_llvm.mlir`. Can have two `// RUN:` there to cover both cases (ftz true/false).